plugins/bookmarks/BookmarksSearchEngine.ts

```diff
@@ -40,7 +41,7 @@ export default class ChromiumBookmarksSearchEngine implements ISearchEngine {
 			icon: findIcon('www'),
 			completion: item.name,
 			event: {
-				type: this.api.environment.platform === 'linux' ? 'linux' : 'unknown',
+				type: 'openUrl',
```

Comment:
suggestion: we should have documentations / comments on event types, expected behavior, etc.

Reply:
we can do it later!